operator/anarchyruntime.py

```diff
@@ -80,7 +80,7 @@ def __init_callback_base_url(self):
             self.callback_base_url = url
             return
         if self.running_all_in_one:
-            self.callback_base_url = 'http://{}:5000'.format(self.anarchy_service_name)
+            self.callback_base_url = 'http://{}:5000'.format(socket.gethostbyname(self.anarchy_service_name))
```

Comment:
Callbacks will be unable to resolve the pod ip by name. So resolve it when setting the callback base url.